### ISSUE TITLE
QLineEdit sometimes doesn't get focus in PyQt5 example on Windows

### ISSUE THREAD

**Issue by cztomczak** [cztomczak/cefpython#404]
QLineEdit = url in navigation bar in qt.py example. This issue occurs only in PyQt5. PySide and PyQt4 do work fine.

Version information:
```cmd
[qt.py] CEF Python 57.0
[qt.py] Python 3.6.3 32bit
[qt.py] PyQt 5.9 (qt 5.9.1)
```

Initially reported on the Forum:
https://groups.google.com/d/topic/cefpython/2kM7foVHlJw/discussion

**Comment by xzycn** [comment]
Hello, this issue happened in PySide2 example too.
Is there a workaround for this? :)

**Comment by cztomczak** [comment]
There are multiple focus issues in CEF on Linux and is tracked by Issue #284. See the comments in that issue for possible resolutions.

**Comment by xzycn** [comment]
@cztomczak ，hi.
Sorry for that didn't clarify clearly: The issue appeared on Windows(Win 10) :).

**Comment by cztomczak** [comment]
The FocusHandler probably should be modified work similarly as in tkinter example. Maybe this will fix the issue. Code:
```py
def OnSetFocus(self, source, **_):
        logger.debug("FocusHandler.OnSetFocus, source={source}"
                     .format(source=source))
        if LINUX:
            return False
        else:
            return True
```

**Comment by cztomczak** [comment]
Fixed in commi 181588c. Fixed it by calling `self.cef_widget.setFocus()` in `FocusHandler.OnGotFocus`.
